### ISSUE TITLE
Won't build on arch linux - fatal error: attr/xattr.h: No such file or directory   

### ISSUE THREAD

**Issue by nihilazo** [rrthomas/plptools#4]
hello,
when I try and build the plptools on my 64-bit arch linux machine, I get this error:
```
fuse.c:23:10: fatal error: attr/xattr.h: No such file or directory
   23 | #include <attr/xattr.h>
      |          ^~~~~~~~~~~~~~
```
I'm not sure what's causing this, but it won't build. I'm following the instructions in the README.

**Comment by rrthomas** [comment]
This is a header file from libattr.

**Comment by nihilazo** [comment]
and how do I get this header file? I have the library, I can't find any other versions of it in the arch repos.

**Comment by rrthomas** [comment]
I don't know, what header files does the library supply in arch? What version is the library?

**Comment by rrthomas** [comment]
OK, this appears to be a change in libattr. Thanks for bringing it to my attention.

**Comment by rrthomas** [comment]
Unfortunately, there isn't a backwards-compatible fix. However, you should be able to get it working quickly by changing the #include that fails to `#include <sys/xattr.h>`.

**Comment by rrthomas** [comment]
I've now pushed a fix; I'd be most grateful if you could test it!

**Comment by rrthomas** [comment]
Thanks for confirming; I've released 1.0.15.
